backend/chainlit/server.py

```diff
@@ -380,13 +380,14 @@ async def oauth_login(provider_id: str, request: Request):
     return response
 
 
-@app.get("/auth/oauth/{provider_id}/callback")
+@app.post("/auth/oauth/{provider_id}/callback")
```

Comment:
This will break other oauth providers, I think you might want to add a `@app.post("/auth/oauth/azure-id/callback")` on top of the existing `@app.get("/auth/oauth/{provider_id}/callback")` route?

Reply:
Thanks for suggestion, I have pushed the changes.